cypress/support/fragments/organizations/organizations.js

```diff
@@ -20,5 +20,29 @@ export default {
     cy.expect(Pane({ id: 'pane-organization-details' }).exists());
     cy.expect(Accordion({ id: summaryAccordionId }).find(KeyValue({ value: organization.name })).exists());
     cy.expect(Accordion({ id: summaryAccordionId }).find(KeyValue({ value: organization.code })).exists());
-  }
+  },
+
+  selectActiveStatus: () => {
+    cy.do(Checkbox('Active').click());
+  },
+
+  checkOrganizationFilter: () => {
+    cy.expect(MultiColumnList({ id: 'organizations-list' }).exists());
+  },
+
+  chooseOrganizationFromList: (organization) => {
+    cy.do(MultiColumnList({ id: 'organizations-list' })
+      .find(MultiColumnListCell({ content: organization.name }))
+      .click());
+  },
+
+  expectcolorFromList: () => {
+    cy.get('.mclRow---e3WhT:first-child').should('have.css', 'background-color', 'rgba(0, 0, 0, 0.08)');
```

Comment:
This code wil not work in other environment because we on different envs we have different class postfixes after --
`.mclRow---e3WhT
`

Reply:
try to use another selector

Reply:
also move this string inside constant with understanding name like
`cons blueColor = 'rgba(0, 0, 0, 0.08)'`